src/main/webapp/app/shared/components/course-exam-archive-button/course-exam-archive-button.component.ts

```diff
@@ -172,7 +172,7 @@ export class CourseExamArchiveButtonComponent implements OnInit, OnDestroy {
     }
 
     canDownloadArchive() {
-        let hasArchive = false;
+        let hasArchive: boolean;
```

Comment:
I think you shouldn't remove default value here

Reply:
I guess it's ok as it is set directly afterwards again.